rules/linux/file_event/file_event_lnx_persistence_cron_files.yml

```diff
@@ -16,20 +17,27 @@ logsource:
     product: linux
     category: file_event
 detection:
-    selection1:
+    selection_1:
         TargetFilename|startswith:
             - '/etc/cron.d/'
             - '/etc/cron.daily/'
             - '/etc/cron.hourly/'
             - '/etc/cron.monthly/'
             - '/etc/cron.weekly/'
             - '/var/spool/cron/crontabs/'
-    selection2:
+    selection_2:
         TargetFilename|contains:
             - '/etc/cron.allow'
             - '/etc/cron.deny'
             - '/etc/crontab'
-    condition: 1 of selection*
+    filter_optional_legit_cron_names:
+        # Note: FPs on docker images: golang, postgres, python, redis, ruby
+        TargetFilename:
+            - '/etc/cron.daily/apt'
+            - '/etc/cron.daily/dpkg'
+            - '/etc/cron.daily/passwd'
+            - '/etc/crontabs/root'
+    condition: 1 of selection* and not 1 of filter_optional_*
 falsepositives:
     - Any legitimate cron file.
 level: medium
```

Comment:
Contemplating about putting this to low. @phantinuss what do you think? How often are cron jobs created on a system. Apart from new software installs...etc.

Reply:
There usually isn't but it's expected that this rule needs environment depending tuning. That's still in the scope of medium imo. We can probably provide more default cronjobs depending on the distribution but most of the jobs should already be on the system when installed. They might get touched during package upgrades nonetheless so having them increases the usefulness of this rule. I would keep it as medium if possible.